perception/mil_vision/ros_tools/colorspaces.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+from __future__ import division, print_function
```

Comment:
This is reasonable. It just means there's one less thing to worry about when the lab inevitably moves to Python 3.